Sidenav first iteration

robbyyt opened the pull request:
Sa te uiti te rog si sa imi spui daca e buna structura ca sa stiu cum continui

robbyyt committed:
Sidenav first iteration

moscugeorge123 commented on src/app/timetable/components/side-menu/side-menu.component.ts:
```diff
@@ -0,0 +1,15 @@
+})
+export class SideMenuComponent implements OnInit {
+
+  constructor() { }
```
contructor and onInit not used

moscugeorge123 commented on src/app/timetable/components/side-menu/side-menu.component.scss:
```diff
@@ -0,0 +1,46 @@
+}
+
+.header-text {
+
```
remove extra space from here

moscugeorge123 commented on src/app/timetable/components/side-menu/side-menu.component.html:
```diff
@@ -0,0 +1,27 @@
+      </div>
+      <div class="menu-container"> 
+          <a mat-button>
+              <mat-icon>schedule</mat-icon>
```
Add routerLink to the links

robbyyt committed:
resolved issues

Sa imi spui pentru celelalte butoane ce rute ar fi pls, clickul e pus doar de test asa, o sa il modifica cand e gata header-ul
